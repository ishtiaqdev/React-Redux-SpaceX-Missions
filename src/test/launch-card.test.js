import React from 'react';
import { configure, shallow } from 'enzyme';
import LaunchCard from '../components/launch-card';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Launch Card should render correctly', () => {
    it('should render correctly in "debug" mode', () => {
      const component = shallow(<LaunchCard debug />);
    
      expect(component).toMatchSnapshot();
    });
});

//Further test can be added here.