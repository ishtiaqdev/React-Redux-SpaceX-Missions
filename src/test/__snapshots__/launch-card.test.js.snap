// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Launch Card should render correctly should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
